maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -44895,7 +44898,21 @@
 /turf/simulated/open/airless,
 /area/horizon/exterior)
 "uZh" = (
-/obj/random/pottedplant,
+/obj/structure/closet/cabinet{
+	name = "beer cabinet"
+	},
+/obj/item/storage/box/fancy/yoke/earthmover,
+/obj/item/storage/box/fancy/yoke/earthmover,
+/obj/item/storage/box/fancy/yoke/ebisu,
+/obj/item/storage/box/fancy/yoke/ebisu,
+/obj/item/storage/box/fancy/yoke/moonlabor,
+/obj/item/storage/box/fancy/yoke/moonlabor,
+/obj/item/storage/box/fancy/yoke/beer,
+/obj/item/storage/box/fancy/yoke/beer,
+/obj/item/storage/box/fancy/yoke/shimauma,
+/obj/item/storage/box/fancy/yoke/shimauma,
+/obj/item/storage/box/fancy/yoke/whistlingforest,
+/obj/item/storage/box/fancy/yoke/whistlingforest,
```

Comment:
add a subtype (e.g. `/obj/structure/closet/cabinet/beer`), add the items to the subtype, and map in the subtype instead.